### DISCUSSION TITLE
Add French and German languages from the European version

### DISCUSSION THREAD

**Discussion by Calvineries** [HarbourMasters/2ship2harkinian#521] [Ideas]
Add French/German languages like on Shipwright from the European version of MM.

**Comment by JoHElB** [comment]
Bonjour calvineries, Je viens de m'inscrire sur Github afin d'essayer d'apporter mon aide.

Je vais essayé d'étudier le sujet et voir si je peux faire une petite traduction Fr du Logiciel.
Ensuite je ferais demande auprès des DEV pour savoir si cela leurs conviens.

J'espère qu'ils vont rendre compatible la Rom EU. 
Zelda MM est de loin mon jeu préféré.

Cdt Johelb
